src/SourceBuild/tarball/content/repos/Directory.Build.targets

```diff
@@ -564,6 +564,56 @@
     <WriteLinesToFile File="$(RepoCompletedSemaphorePath)RemoveBuiltPackagesFromCache.complete" Overwrite="true" />
   </Target>
 
+  <Target Name="DirSizeBeforeBuild"
+          BeforeTargets="Build"
+          Condition=" '$(CleanWhileBuilding)' == 'true' ">
+    <Exec Command="echo 'DirSize Before Building $(RepositoryName)' >> $(LoggingDir)DirSize.log" />
```

Comment:
Curious the value in capturing this in a separate log file vs the normal build output/log.  I realize at this point in time it may be more convenient but once this problem is fixed I wonder if it would get more visibility in the regular build logs.

Reply:
I woke up this morning thinking the same thing.  Fixed.